draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -788,33 +768,53 @@ Content-Type: text/plain
-   requirements for the "https" scheme, except that TCP port 443 is the
-   default if the port subcomponent is empty or not given,
-   and the user agent &MUST; ensure that its connection to the origin
-   server is secured through the use of strong encryption, end-to-end,
-   prior to sending the first HTTP request.
+   identifiers within a hierarchical namespace governed by a potential origin
+   server capable of establishing a secure and verifiable TLS connection on
+   behalf of the identified authority (<xref target="RFC8446"/>).
 </t>
 <sourcecode type="abnf7230"><iref primary="true" item="Grammar" subitem="https-URI"><!--terminal production--></iref>
-  <x:ref>https-URI</x:ref> = "https:" "//" <x:ref>authority</x:ref> <x:ref>path-abempty</x:ref> [ "?" <x:ref>query</x:ref> ]
+  <x:ref>https-URI</x:ref> = "https" "://" <x:ref>authority</x:ref> <x:ref>path-abempty</x:ref> [ "?" <x:ref>query</x:ref> ]
 </sourcecode>
 <t>
-   Note that the "https" URI scheme depends on both TLS and TCP for
-   establishing authority.
-   Resources made available via the "https" scheme have no shared
-   identity with the "http" scheme even if their resource identifiers
-   indicate the same authority (the same host listening to the same
-   TCP port).  They are distinct namespaces and are considered to be
-   distinct origin servers.  However, an extension to HTTP that is
-   defined to apply to entire host domains, such as the Cookie protocol 
-   <xref target="RFC6265"/>, can allow information
-   set by one service to impact communication with other services
-   within a matching group of host domains.
+   The origin server for an "https" URI is identified by the
+   <x:ref>authority</x:ref> component, which includes a host identifier
+   and optional port number (<xref target="RFC3986" x:fmt="," x:sec="3.2.2"/>).
+   If the port subcomponent is empty or not given, TCP port 443
+   (the reserved port for HTTP over TLS) is the default.
+</t>
+<t>
+   The hierarchical path component and optional query component identify the
+   target resource within that origin server's name space.
+</t>
+<t>
+   A sender &MUST-NOT; generate an "https" URI with an empty host identifier.
+   A recipient that processes such a URI reference &MUST; reject it as invalid.
+</t>
+<t>
+   When an "https" URI is used within a context that calls for access to the
+   indicated resource, a client &MAY; attempt access by resolving the host
+   identifier to an IP address, establishing a TCP connection to that address
+   on the indicated port, securing the connection end-to-end by successfully
+   initiating TLS over TCP, and sending an HTTP request message to the server
+   over that secured connection containing the URI's identifying data
+   (<xref target="http.message"/>).
+</t>
+<t>
+   The client &MUST; ensure that its connection to the origin server is
+   secured through the use of strong encryption, end-to-end, prior to sending
+   the first HTTP request.
```

Comment:
"secured" again is a little vague, though you do define it, it's not enough to use encryption.  Maybe instead say that the client `&MUST;` ensure that all requests it sends and all responses it accepts are only sent to a server that it authenticated as being authoritative and that those messages have confidentiality and integrity protection.

Reply:
I tried several variations and finally had to go with an explicit definition for "secured" in this context. Trying to detail it three times in the same section was hopeless.